### PR TITLE
Fix syntax errors in the document

### PR DESCRIPTION
**Pull request by Fish-pro**
Signed-off-by: Fish-pro <zechun.chen@daocloud.io>

**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by RainbowMango** [comment]
@Tingtal Could you please take a look?

File: docs/proposals/hpa/federated-hpa.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ this KEP.  Describe why the change is important and the benefits to users.
 1. Bring HPA from single cluster to multiple clusters.
 1. Compatible with the HPA related resources in the single cluster.
 1. Tolerate the disaster of member cluster or karmada control plane.
-1. It is better to integrate well with the scenarios such as workloads shifting and cloud burst.
+1. It is better to integrate well with the scenarios such as workloads shifting and cloud eruption.
```
Comment (Tingtal) [inline docs/proposals/hpa/federated-hpa.md:34]: "shifting" and "eruption" are not the terms commonly used in cloud computing.
Consider "migration" and "scaling".
"cloud bursting" is a fixed term too, but it relates to on-prem infra.

**Review by Tingtal — CHANGES_REQUESTED** [review]
Language checks.

File: docs/proposals/hpa/federated-hpa.md (hunk 1 above)
Comment (Fish-pro) [reply docs/proposals/hpa/federated-hpa.md:34]: thanks for tips, I got it

**Comment by Fish-pro** [comment]
@Tingtal Thanks to careful checking, modifications have been applied

**Comment by Tingtal** [comment]
> @Tingtal Thanks to careful checking, modifications have been applied

Always welcome. 😃

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**